docs/CONTRIBUTING.md

```diff
@@ -626,31 +626,33 @@ have appeared in the `built-distribution` directory.
 
 The IDE will connect to the running project manager to look up the project and
 start the language server. The required version of the language server is
-specified in the `enso-version` field of the `project.yaml` project description
-(Enso projects are located in the `~/enso` directory).
+specified in the `edition` field of the `package.yaml` project description (Enso
+projects are located in the `~/enso` directory).
```

Comment:
Is this true on all OSes? Also Windows?

Reply:
Why would it not work on all OSes? I mean, the question of Michael is very important, however, if by any accident the answer would be "no", then I'd consider that as a serious bug in our config system.